lib/tools/settings-client-commands.js

```diff
@@ -247,7 +247,7 @@ commands.setGeoLocation = async function setGeoLocation (location, isEmulator =
     await this.adbExec(['emu', 'geo', 'fix', ...(args.map((arg) => arg.replace('.', ',')))]);
   } else {
     const args = [
-      'am', 'startservice',
+      'am', await this.getApiLevel() >= 26 ? 'start-foreground-service' : 'startservice',
```

Comment:
it would probably make sense to enclose `await this.getApiLevel() >= 26` into parentheses for better readablity

Reply:
Thank you for the review. Revised by https://github.com/appium/appium-adb/pull/603/commits/39e90b7afd04bd4f24176794e0ebf1b76bc0dd2e.